packages/core/src/__tests__/document.test.ts

```diff
@@ -84,7 +85,7 @@ jest.mock('../dispatcher', () => {
       retrieveRecord: jest.fn(cid => {
         return recs[cid.toString()]
       }),
-      retrieveFromIPFS: jest.fn((cid, path) => {
+      retrieveFromIPFS: jest.fn((cid) => {
```

Comment:
The function signature being mocked here does take two arguments, the second being `path`.  Since the mock implementation here doesn't actually use the second arg, maybe call it `pathUnused` or something like that?  But I think it makes sense to keep the second arg here so it matches the signature of the function being mocked

Reply:
I'll just get it back.